Makefile

```diff
@@ -26,7 +26,18 @@ host-hlint: ## Run the system (not Nix) version of hlint
 	hlint bench cardano-{api,cli,node,node-capi,node-chairman,submit-api,testnet,tracer}
 
 stylish-haskell: ## Apply stylish-haskell on all *.hs files
-	@find . -type f -name "*.hs" -not -path '.git' -print0 | xargs -0 stylish-haskell -i
+	@! find .							\
+	       -name '\\.*' -prune					\
+	    -o -name 'dist-newstyle' -prune				\
+	    -o -name '*.hs'						\
+	       -execdir sh -c "realpath {}				\
+			| xargs stylish-haskell				\
+				-c $(CURDIR)/.stylish-haskell.yaml	\
+				-i"					\
+			\;						\
+			3>&1 1>&2 2>&3					\
+	| grep .	3>&1 1>&2 2>&3
```

Comment:
It seems to me this is complicated because we want to support running from elsewhere than the repository root.

Could we assume we run from the repository root and go with something simpler like `stylish-haskell -i $(git ls-files "*.hs")` (I'm assuming here that `stylish-haskell` will pick up the config file in the working directory).

Reply:
> It seems to me this is complicated because we want to support running from elsewhere than the repository root.
> 
> Could we assume we run from the repository root and go with something simpler like `stylish-haskell -i $(git ls-files "*.hs")`? (I'm assuming here that `stylish-haskell` will pick up the config file in the working directory).

Please give me an idea of the pattern of invocation involved with running from elsewhere besides the repository root. I'm cooking up a fixup for this for your review, but the more information the better. I might get something out for you before you get back to me, but no worries, I'll change it up again to take your answer & further review into account.

Reply:
It's probably a good idea for me to restart the commit series around an adaptation of your code to makefile rules as a starting point. At minimum, that stays out of the weeds enough for other people to understand it. It may be worth starting a fresh branch so the one the PR is now attached to can be there to look at until the thought of cleaning up the execution failure up and whatever else is wrong with the makefile rule gets stale enough to delete it.

Beyond that, I can layer some things atop that in further commits in the series for people to see if they want anything more out of it if I get back to it. I'll also credit you as co-author in the initial commit corresponding to your code above.

Reply:
I dropped in a total rewrite to your suggestion. The very last commit is the thing to check. The commit history is there mostly for people to look at the ideas that bubbled around. It might be worth looking commit-by-commit after some point in the series to track what's ongoing with it all.

Reply:
I'm resolving this conversation since your last implementation is fine with me, and so I'm approving :heavy_check_mark: . I suggest your squash all commits in the end, before merging.